scripts/docker/Dockerfile.clang

```diff
@@ -1,31 +1,19 @@
 FROM nvidia/cuda:9.2-devel
 
-RUN apt-get update && apt-get install -y \
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && \
+    apt-get install -y apt-transport-https ca-certificates gnupg software-properties-common wget && \
+    wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null && \
+    apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(cat /etc/lsb-release | grep DISTRIB_CODENAME | awk -F '=' '{print $NF}') main" && \
```

Comment:
there is no need to `cat` the file
`grep DISTRIB_CODENAME /etc/lsb-release | cut -d= -f2` does what you want